undertow/src/main/java/org/wildfly/extension/undertow/ConsoleAccessLogDefinition.java

```diff
@@ -123,6 +122,10 @@ protected void performRuntime(final OperationContext context, final ModelNode op
             serviceBuilder.setInstance(service)
                     .setInitialMode(ServiceController.Mode.ACTIVE)
                     .install();
+            } catch (RuntimeException e) {
+                e.printStackTrace();
+                throw e;
+            }
```

Comment:
@pferraro This looks like leftover debugging code.

Reply:
Indeed. Will fix.